docs/source/errors.md

```diff
@@ -53,6 +53,12 @@ If Apollo Gateway encounters an error, composition fails. This document lists co
 | `REQUIRES_FIELDS_MISSING_ON_BASE` | The `fields` argument of an entity field's `@requires` directive includes a field that is not defined in the entity's originating subgraph.`|
 | `REQUIRES_USED_ON_BASE` | An entity field is marked with `@requires` in the entity's originating subgraph, which is invalid. |
 
+## `@tag`
+
+| Code | Description |
+|---|---|
+| `TAG_USED_WITH_EXTERNAL` | Fields marked as `@external` cannot also have `@tag` usages.  |
```

Comment:
This seems fine for now, but I wonder if we should generalize this to other directives.

Reply:
@martijnwalraven @trevor-scheer should we add the same handling for`@inaccessible` here as well -- even if we don't generalize just yet?

Reply:
also how will we handle `@tag` and `@inaccessible` on value types referenced only from `@external` fields, like https://github.com/apollographql/supergraph-demo/pull/165/files#diff-4a1edface097b589a87522e4fc919429284c343628e3bca83170a6db5e1bc678R9?

Reply:
@prasek Included inaccessible in this PR, see commit 3d51e58

Going to handle value type concerns in a separate PR for the sake of timing.